lib/rubocop/cop/style/align_parameters.rb

```diff
@@ -20,6 +17,26 @@ def on_send(node)
           check_alignment(args, base_column(node, args))
         end
 
+        def on_def(node)
```

Comment:
We have a mixin named `OnMethodDef` that you can use to avoid some duplication here for `def` and `defs` nodes.